manifests/params.pp

```diff
@@ -122,7 +133,7 @@
   # Owner of the environments dir: for cases external service needs write
   # access to manage it.
   $server_environments_owner   = $user
-  $server_environments_group   = 'root'
+  $server_environments_group   = $root_group
```

Comment:
I would think undef because, generally speaking, you will want it to inherit permissions from it's parent folder. There are also some interesting issues with permissions on Windows via Puppet outlined at https://docs.puppetlabs.com/puppet/3.6/reference/resources_file_windows.html